frontend/packages/console-app/src/components/dashboards-page/console-plugins-health-resource/ConsolePluginsPopover.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { Stack, StackItem } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import { ResourcesObject, WatchK8sResults, WatchK8sResources } from '@console/dynamic-plugin-sdk';
+import { ConsoleOperatorConfigModel, ConsolePluginModel } from '@console/internal/models';
+import { ConsolePluginKind, referenceForModel } from '@console/internal/module/k8s';
+import { isLoadedDynamicPluginInfo } from '@console/plugin-sdk/src';
+import { useDynamicPluginInfo } from '@console/plugin-sdk/src/api/useDynamicPluginInfo';
+import { StatusPopupSection } from '@console/shared/src/components/dashboard/status-card/StatusPopup';
+
+const ConsolePluginsPopover: React.FC<ConsolePluginsPopoverProps> = ({ consolePlugins }) => {
+  const { t } = useTranslation();
+  const [pluginInfoEntries] = useDynamicPluginInfo();
+  const loadedPlugins = pluginInfoEntries.filter(isLoadedDynamicPluginInfo);
+  const enabledPlugins = loadedPlugins.filter((plugin) => plugin.enabled === true);
+
+  return (
+    <Stack hasGutter>
+      {/* TODO: get correct description text */}
+      <StackItem>{t('console-app~A brief description about dynamic plugins.')}</StackItem>
+      <StackItem>
+        <StatusPopupSection
+          firstColumn={t('console-app~Installed plugins')}
+          secondColumn={
+            <>
+              {t('console-app~{{enabledCount}}/{{totalCount}} enabled', {
+                enabledCount: enabledPlugins.length,
+                totalCount: consolePlugins.data.length,
+              })}
+            </>
+          }
+        >
+          <Link
+            to={`/k8s/cluster/${referenceForModel(
+              ConsoleOperatorConfigModel,
+            )}/cluster/console-plugins`}
+          >
+            {t('console-app~View all')}
+          </Link>
+        </StatusPopupSection>
+      </StackItem>
+    </Stack>
+  );
+};
+
+/** Used in extension */
+export const consolePluginsResources: WatchK8sResources<ResourcesObject> = {
```

Comment:
```suggestion
export const consolePluginsResources: WatchK8sResources<WatchConsolePluginsResource> = {
```

You have the type already defined below here -- no need to use the general one.